README.md

```diff
@@ -220,16 +225,15 @@ A JSON string containing additional metadata about the log line that is passed.
 * Default: `5`
 * Example Values: `logging,golang`
 
+Tags to be added to each message.
 
-List of tags used to dynamically group hosts.
-
-#### Key
+#### Timestamp
 
 * _**Required**_
-* Type: `string`
-* Example Values: `123abc`
+* Type: `time.Time`
+* Example Values: `time.Now()`
 
-The [LogDNA Ingestion Key](https://app.logdna.com/manage/profile) associated with your account.
+Epoch ms time to use if not provided elsewhere.
```

Comment:
I think this should be marked optional, and we should include `Date.now()` as the default rather than an example.

also perhaps:

`Epoch ms time to use for the current message`

Reply:
So this timestamp here is not linked to the current message while this timestamp here is: https://github.com/logdna/logdna-go/pull/14/files#diff-04c6e90faac2675aa89e2176d2eec7d8R387

Reply:
Also default is set to time.now() in the default here: https://github.com/logdna/logdna-go/blob/master/logger/transport.go#L103

Reply:
supplying a timestamp in `Options` _only_ makes sense in the context of a message. it would not make sense to set a Logger-level timestamp (every message would be logged at the same timestamp)

Reply:
agreed- but in the current state of the client we support the ability for a user to set a Logger-level timestamp

Reply:
Let's fix that then, rather than continuing to document how to get broken behavior. This would be super confusing for a user and it will be difficult to change the behavior later without a breaking change. That can be a separate card IMO.

Reply:
![Screen Shot 2020-09-23 at 11 17 27 AM](https://user-images.githubusercontent.com/10788080/94052915-79457580-fd8e-11ea-841d-0b7a65c06ef6.png)
![Screen Shot 2020-09-23 at 11 17 33 AM](https://user-images.githubusercontent.com/10788080/94052922-7b0f3900-fd8e-11ea-9c27-98c6f549411d.png)

Reply:
yeah we gotta fix that. you shouldn't be able to set a timestamp on the logger itself, it should only come from 1) `LogWithOptions` or 2) automatically set when generating the line (which is the most common use-case)

Reply:
Am going to address this issue in a separate Jira ticket/PR.